snapstack/scripts/etc/snap-cinder/cinder/cinder/cinder.conf.d/database.conf

```diff
@@ -0,0 +1,2 @@
+[database]
+connection = mysql+pymysql://cinder:cinder@localhost/cinder
```

Comment:
I think all the config files should live in their corresponding snap's test directory and be pulled in by snapstack.